.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "partiql-conformance-tests/partiql-tests"]
 	path = partiql-conformance-tests/partiql-tests
 	url = https://github.com/partiql/partiql-tests.git
+[submodule "partiql-playground/ace-builds"]
```

Comment:
We have an option of using `npm` but I thought this is an overkill at this stage as we might move the front-end code out of this package, hence decided to use `submodule` for now. Happy to discuss this further if `npm` is favorable.

Reply:
Just curious what would be involved to use `npm` rather than a git submodule?

Reply:
If not mistaken, we need to define the project as a `Node.js` project and then change the entry point to node like `app.js`.